### PR TITLE
feat: add simple linking to the examples

### PR DESCRIPTION
**Pull request by mbwhite**
Simple description of the examples and a link to the new one added in the substrait-java repo; this will need to wait until after that is merged to ensure no broken links.,

### CODE REVIEW

File: site/docs/tutorial/examples.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Code samples and examples
+
+It's very useful to have examples of how APIs are used; both to get information on teh best practicses for using APIs and ideas of how they can be used.
```
Comment (jacques-n) [inline site/docs/tutorial/examples.md:3]: teh

Comment (mbwhite) [reply site/docs/tutorial/examples.md:3]: thanks @jacques-n   when the PR in the substrait-java is merged.. I'll update this one

File: site/docs/tutorial/examples.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# Code samples and examples
+
+It's very useful to have examples of how APIs are used; both to get information on the best practices for using APIs and ideas of how they can be used. 
+
+Each language binding is intended to contain examples that are relavent to that language. Hoped that this list will grow over time, new submissions always welcome.
```
Comment (bvolpato) [inline site/docs/tutorial/examples.md:5]:
```suggestion
Each language binding is intended to contain examples that are relevant to that language. New contributions are always welcome.
```

File: site/docs/tutorial/examples.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+
+## Java
+
+- [Substrait-Spark](https://github.com/substrait-io/substrait-java/tree/main/examples/substrait-spark) this demonstrates how Substrait plans can be created and consumed within Apache Spark. The examples run within a simple Spark cluster of a couple of docker containers.
```
Comment (bvolpato) [inline site/docs/tutorial/examples.md:9]:
```suggestion
- [Substrait-Spark](https://github.com/substrait-io/substrait-java/tree/main/examples/substrait-spark) this demonstrates how Substrait plans can be created and consumed within Apache Spark. The examples run within a simple Spark cluster, composed by a couple of Docker containers.
```

**Review by bvolpato — APPROVED** [review]
LGTM, just a few nit suggestions / ideas to make it clearer

**Comment by mbwhite** [comment]
thanks @bvolpato I've accepted those changes

**Review by EpsilonPrime — APPROVED** [review]

**Review by westonpace — APPROVED** [review]

**Pull request merged**